test/tstQueryTreeComparisonWithBoost.cpp

```diff
@@ -132,11 +132,11 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(boost_rtree_spatial_predicate, TreeTypeTraits,
   auto points = ArborXTest::make_random_cloud<ArborX::Point>(
       ExecutionSpace{}, n_points, Lx, Ly, Lz);
 
-  Kokkos::View<double *, ExecutionSpace> radii("radii", n_points);
+  Kokkos::View<float *, ExecutionSpace> radii("radii", n_points);
   auto radii_host = Kokkos::create_mirror_view(radii);
   // use random radius for the search
   std::default_random_engine generator;
-  std::uniform_real_distribution<double> distribution_radius(
+  std::uniform_real_distribution<float> distribution_radius(
```

Comment:
What about leaving the template parameter out and passing `0.f, std::sqrtf(...)`?
(Don't feel to strongly about it)

Reply:
We had problems with omitting template arguments for uniform distributions for CUDA 11.0-11.3 in #850.